src/app/not-found.tsx

```diff
@@ -0,0 +1,37 @@
+import Image from "next/image";
+import Link from "next/link";
+import errIcon from "../../assets/icons/404_Icon.svg";
+
+export default function NotFound() {
+  return (
+    <div className="bg-[#37526B]">
+      <div className="mx-auto w-full max-w-[1440px] pt-[90px] text-center">
+        <div className="mb-[38px]">
+          <Image
+            className="mx-auto h-[242px] w-[242px]"
+            src={errIcon}
+            alt="404 Error Icon"
+            priority
+          />
```

Comment:
Using the Tailwind classes makes sense, but it's causing the unit test error. I'd suggest we just apply the `width` and `height` props directly and remove the classes for those two attributes.

e.g. 

```suggestion
          <Image
            className="mx-auto"
            src={errIcon}
            alt="404 Error Icon"
            width={242}
            height={242}
            priority
          />
```